src/main/java/it/uniroma2/pellegrini/z64sim/controller/SettingsController.java

```diff
@@ -204,6 +205,14 @@ private Settings() {
 
         protected static Settings loadConfiguration() throws SettingsException {
             Settings settings;
+            
+            // Following XDG standard if on *NIX-like
+            if(SystemUtils.IS_OS_LINUX || SystemUtils.IS_OS_UNIX) {
+                configurationDirectoryPath = System.getProperty("user.home") + "/.config/z64sim";
+            } else {
+                configurationDirectoryPath = System.getProperty("user.home") + System.getProperty("file.separator") + ".z64sim";
```

Comment:
To make it fully portable, we should check for `%APPDATA%` on Windows. If not set, it should fallback on `C:\Documents and Settings\{username}\Application Data` for XP and previous and `C:\Users\{username}\AppData\Roaming` for later versions.

On MacOS, this should be `~/Library/Application Support/`.

Reply:
Done an enum with those specifics for win/mac/*nix